src/mumble/GlobalShortcut_win.h

```diff
@@ -134,6 +134,8 @@ public slots:
 	std::unique_ptr< GKeyLibrary > m_gkey;
 #endif
 	DeviceMap::iterator addDevice(const HANDLE deviceHandle);
+
+	static std::string utf16To8(const std::wstring &wstr);
```

Comment:
Would it make sense to move this to a utility class in case we want to use it later in other parts of the code base?

Reply:
It's Windows-only. The only other instance of `WideCharToMultiByte()` is:

https://github.com/mumble-voip/mumble/blob/b75fe546ab6e5f32204e20df7ee3ceb8ce70262c/src/ProcessResolver.cpp#L67-L76

Which, now that I'm reading it, should be refactored to avoid the use of `MAX_PATH`.

I think I'll do that in a separate pull request (while also creating a class to avoid code duplication).

Reply:
I don't see a reason for not doing that right away.

Reply:
`utf16To8()` now lives in `win.h`, `win` namespace.

I will create a separate pull request to refactor `ProcessResolver.cpp`.